packages/features/bookings/lib/get-booking.ts

```diff
@@ -109,24 +109,20 @@ export const getBookingWithResponses = <
 export default getBooking;
 
 export const getBookingForReschedule = async (uid: string) => {
-  let eventTypeId: number | null = null;
   let rescheduleUid: string | null = null;
-  eventTypeId =
-    (
-      await prisma.booking.findFirst({
-        where: {
-          uid,
-        },
-        select: {
-          eventTypeId: true,
-        },
-      })
-    )?.eventTypeId || null;
+  const eventType = await prisma.booking.findFirst({
```

Comment:
Shouldn't this be booking then?

```suggestion
  const booking = await prisma.booking.findFirst({
```

Reply:
yeahp, thanks!